<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_page</class>
 <widget class="QWidget" name="User_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Page</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="searchLayout">
     <item>
      <widget class="QLineEdit" name="searchItemInput">
       <property name="placeholderText">
        <string>Search by item name or category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchItemBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Search Items</string>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="categoryTabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGroceries">
      <attribute name="title">
       <string>Groceries</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutGroceries">
       <item>
        <widget class="QTableWidget" name="groceriesTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabElectronics">
      <attribute name="title">
       <string>Electronics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutElectronics">
       <item>
        <widget class="QTableWidget" name="electronicsTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabLiquors">
      <attribute name="title">
       <string>Liquors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutLiquors">
       <item>
        <widget class="QTableWidget" name="liquorsTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSmokingProducts">
      <attribute name="title">
       <string>Smoking Products</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutSmoking">
       <item>
        <widget class="QTableWidget" name="smokingTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCosmetics">
      <attribute name="title">
       <string>Cosmetics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutCosmetics">
       <item>
        <widget class="QTableWidget" name="cosmeticsTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabClothes">
      <attribute name="title">
       <string>Clothes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutClothes">
       <item>
        <widget class="QTableWidget" name="clothesTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabFurniture">
      <attribute name="title">
       <string>Furniture</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayoutFurniture">
       <item>
        <widget class="QTableWidget" name="furnitureTable">
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="addToCartBtn">
       <property name="text">
        <string>Add to Cart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="quantityInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="increaseQtyBtn">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="decreaseQtyBtn">
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="checkoutBtn">
       <property name="text">
        <string>Checkout</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewCartBtn">
       <property name="text">
        <string>View Cart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="feedbackBtn">
       <property name="text">
        <string>Feedback</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsBtn">
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutLogoutRefresh">
     <item>
      <widget class="QPushButton" name="refreshBtn">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="logoutBtn">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Logout</string>
       </property>A
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
